core/src/main/java/tech/tablesaw/io/DataFrameReader.java

```diff
@@ -115,4 +122,12 @@ public Table db(ResultSet resultSet, String tableName) throws SQLException {
     public Table html(String url) throws IOException {
         return new HtmlTableReader().read(url);
     }
+    
+    public List<Table> xlsx(XlsxReadOptions options) throws IOException {
```

Comment:
Maybe here, or perhaps in the docs, we should document `Requires that "org.apache.poi:poi-ooxml" be on your classpath`